React/CoreModules/RCTAlertController.m

```diff
@@ -20,7 +20,18 @@ @implementation RCTAlertController
 - (UIWindow *)alertWindow
 {
   if (_alertWindow == nil) {
-    _alertWindow = [[UIWindow alloc] initWithFrame:RCTSharedApplication().keyWindow.bounds];
+    if (@available(iOS 13.0, *)) {
+        for (UIWindowScene *scene in RCTSharedApplication().connectedScenes) {
+          if(scene.activationState == UISceneActivationStateForegroundActive && scene.class == UIWindowScene.class) {
+            _alertWindow = [[UIWindow alloc] initWithWindowScene:scene];
+            break;
+          }
+        }
+      }
+      if (_alertWindow == nil) {
+        _alertWindow = [[UIWindow alloc] initWithFrame:RCTSharedApplication().keyWindow.bounds];
+      }
```

Comment:
```suggestion
      } else {
        _alertWindow = [[UIWindow alloc] initWithFrame:RCTSharedApplication().keyWindow.bounds];
      }
```

Reply:
you already do this check few lines above

Reply:
@ronal2do I think what he meant was if somehow the _alertWindow is still null, it should attempt to use the old way